utils/certs/gen_certificates.sh

```diff
@@ -167,21 +172,40 @@ function generate_server_cert () {
     ${CERTS}/server.crt"
 }
 
+function populate_clients_dir () {
+    cp ${CERTS}/admin.crt         ${CLIENTS}/admin.crt
+    chown daos_server.daos_server ${CLIENTS}/admin.crt
+    chmod 640                     ${CLIENTS}/admin.crt
+    cp ${CERTS}/agent.crt         ${CLIENTS}/agent.crt
```

Comment:
(lint) Double quote to prevent globbing and word splitting. [SC2086]
(lint) Double quote to prevent globbing and word splitting. [SC2086]